src/vos/vos_common.c

```diff
@@ -55,6 +55,37 @@ vos_tls_get(void)
 #endif /* VOS_STANDALONE */
 }
 
+#ifdef VOS_STANDALONE
+int
+vos_profile_start(char *path, int avg)
+{
+	struct daos_profile *dp;
+	int rc;
+
+	if (standalone_tls == NULL)
+		return 0;
+
+	rc = daos_profile_init(&dp, path, avg, 0, 0);
+	if (rc)
+		return rc;
+
+	standalone_tls->vtl_dp = dp;
+	return 0;
+}
+
+void
+vos_profile_stop()
+{
+	if (standalone_tls == NULL || standalone_tls->vtl_dp == NULL)
+		return;
+
+	daos_profile_dump(standalone_tls->vtl_dp);
+	daos_profile_destroy(standalone_tls->vtl_dp);
+	standalone_tls->vtl_dp = NULL;
+}
+
```

Comment:
minor: srv_profile_start/stop() & vos_profile_start/stop() looks bit redundant to me, can it be unified to daos_profile_start/stop()?